src/StoreApi/Utilities/OrderController.php

```diff
@@ -44,6 +44,9 @@ public function create_order_from_cart() {
 	 * @param \WC_Order $order The order object to update.
 	 */
 	public function update_order_from_cart( \WC_Order $order ) {
+		// Ensures Local pickups are accounted for.
+		add_filter( 'woocommerce_order_get_tax_location', array( $this, 'handle_local_pickup_taxes' ) );
```

Comment:
We want to limit where we call this logic, this function matters because this is where we call `calculate_totals`.